packs/orion/tests/test_action_node_remanage.py

```diff
@@ -0,0 +1,108 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+import yaml
+from mock import Mock, MagicMock
+
+from st2tests.base import BaseActionTestCase
+
+from node_remanage import NodeRemanage
+
+__all__ = [
+    'NodeRemanageTestCase'
+]
+
+
+class NodeRemanageTestCase(BaseActionTestCase):
+    action_cls = NodeRemanage
+
+    def test_run_no_config(self):
+        self.assertRaises(ValueError,
+                          NodeRemanage,
+                          yaml.safe_load(
+                              self.get_fixture_content('blank.yaml')))
+
+    def test_run_is_instance(self):
```

Comment:
Thanks for this suggestion, I've refactored and it's made things more readable (`277 insertions(+), 720 deletions(-)`).